.clang-format

```diff
@@ -59,3 +59,15 @@ IndentPPDirectives: AfterHash
 
 # Include blocks style
 IncludeBlocks: Preserve
+
+AttributeMacros:
+  - OPENTELEMETRY_UNLIKELY
+  - OPENTELEMETRY_LIKELY
+  - OPENTELEMETRY_MAYBE_UNUSED
+  - OPENTELEMETRY_DEPRECATED
+  - OPENTELEMETRY_API_SINGLETON
+  - OPENTELEMETRY_LOCAL_SYMBOL
+  - OPENTELEMETRY_EXPORT
+  - OPENTELEMETRY_SANITIZER_NO_MEMORY
+  - OPENTELEMETRY_SANITIZER_NO_THREAD
```

Comment:
Can we move this formatting in separate PR. I don't think it is related to deprecating event logger.

Reply:
I found that clang-format doesn't recognize our attribute macros as attributes but sometimes as class names, which will make clang-format make strange style formatting for this PR. So my suggestion is including these settings in this PR to reduce confuse.